templates/collectd-openvpn.conf.j2

```diff
@@ -0,0 +1,6 @@
+LoadPlugin openvpn
+<Plugin openvpn>
+    StatusFile "{{ openvpn_status_file_path }}"
```

Comment:
can use existing variables to avoid redundancy ... 
can replace with this -> "{{ openvpn_etcdir }}/{{ openvpn_status }}"